deploy/objects/class.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: managed-nfs-storage
-provisioner: fuseim.pri/ifs # or choose another name, must match deployment's env PROVISIONER_NAME'
+provisioner: nfs-subdir-external-provisioner # or choose another name, must match deployment's env PROVISIONER_NAME'
```

Comment:
Just thinking out loud, should we rename `fuseim.pri/ifs` to `k8s-sigs.io/nfs-subdir-external-provisioner`? WDYT @yonatankahana

Reply:
it sounds correct, it follows the namespace convention as shown in the [storage class concept](https://kubernetes.io/docs/concepts/storage/storage-classes/).

Reply:
done.